schemas/did-uuid.js

```diff
@@ -12,11 +12,15 @@ const cfg = config['veres-one-validator'];
 const prefix = cfg.environment === 'test' ? 'did:v1:test' : 'did:v1';
 /* eslint-disable-next-line max-len */
 const pattern = `^${prefix}:uuid:([a-fA-F0-9]{8}-[a-fA-F0-9]{4}-4[a-fA-F0-9]{3}-[89abAB][a-fA-F0-9]{3}-[a-fA-F0-9]{12})$`;
+const expectedLength = cfg.environment === 'test' ? 53 : 48;
 
 const schema = {
   title: 'DID UUID',
   type: 'string',
   pattern,
+  // prefix can be 11 chars or 6 chars for a total of 53 or 48
```

Comment:
This comment here is confusing.  Perhaps move above the expectedLength calculation?

Reply:
I moved this earlier today. Not sure which commit it is.